source/develop/release-management/features/network/isolated-ports.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Isolated ports
+category: feature
+authors: dholler
+feature_name: Isolated ports
+feature_modules: engine,network
+feature_status: In Development
+---
+
+# Isolated ports
+
+## Summary
+
+This feature prevents communication of VMs and even visibility between VMs
```

Comment:
`VMs inter-communication` maybe? Or is it also with the outside? I have no idea how this feature works, so these are real questions. The new wording is better but I think it may be improved further.

Reply:
I tried to improve further.